createwiki.sh

```diff
@@ -48,6 +48,11 @@ php $PATCHDEMO/wikis/$NAME/w/maintenance/install.php \
 # apply our default settings
 cat $PATCHDEMO/LocalSettings.txt >> $PATCHDEMO/wikis/$NAME/w/LocalSettings.php
 
+# Add the proxy if selected
+if [ "${USE_PROXY}" = "1" ] then
+	cat $PATCHDEMO/wikis/$NAME/w/LocalSettings.php $PATCHDEMO/LocalSettings-proxy.txt
```

Comment:
Missing `>>`?

Reply:
and they're the wrong way around
`cat txt >> php`

Reply:
(done)